bin/cli.js

```diff
@@ -15,19 +15,23 @@ if (argv.register) {
   exit(boxen('\u2714 KEY stored!', {align: 'center', borderColor: 'green', padding: 1, margin: 1}), 0)
 }
 
+var googleTranslate = require('google-translate')(API_KEY)
+
 if (argv.st) {
-  config.setTargetLang(argv.st)
+  config.setTargetLang(argv.st, googleTranslate)
   exit(boxen('\u2714 Target lang set!', {align: 'center', borderColor: 'green', padding: 1, margin: 1}), 0)
 }
 
-var googleTranslate = require('google-translate')(API_KEY)
-
 googleTranslate.translate(argv.text, argv.source, argv.target || config.getTargetLang(), function (err, translation) {
   if (err) return exitWithError(err)
   var translatedText = translation.translatedText
   var sourceLanguage = translation.detectedSourceLanguage || argv.source
-  var targetLanguage = translation.targetLanguage
-  console.log(boxen(`${sourceLanguage} \u2192 ${targetLanguage}\n\n${translatedText} `, {align: 'center', borderColor: 'green', padding: 1, margin: 1}))
+  var targetLanguage = translation.targetLanguage || argv.target
+  if (argv.b)
+    console.log(translatedText)
```

Comment:
If this is for `stdout` usage purpose. Keep in mind that `console.log` will happend a `\n` to the end of the line. Maybe it's better to `process.stdout.write(translatedText)`.

Reply:
Yeah, I did notice that in the tests. Great, I'll fix that.